src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -0,0 +1,148 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Internationalize from 'box-ui-elements/es/elements/common/Internationalize';
+import {
+    dateCellRenderer,
+    sizeCellRenderer,
+    fileNameCellRenderer,
+} from 'box-ui-elements/es/features/virtualized-table-renderers';
+import VirtualizedTable from 'box-ui-elements/es/features/virtualized-table';
+import { Column } from 'react-virtualized/dist/es/Table/index';
+
+class ArchiveExplorer extends React.Component {
+    static propTypes = {
+        data: PropTypes.arrayOf(
+            PropTypes.shape({
+                type: PropTypes.string,
```

Comment:
Are all of these properties optional? If not, should some be marked with `isRequired`?